docs/source/changelog.rst

```diff
@@ -1,6 +1,21 @@
 Changelog
 =========
 
+1.21.2 - 2018-03-05
+-------------------
+
+-  Fix bug where we didn't check idle/saturated when stealing (:pr:`1801`) `Matthew Rocklin`_
+-  Fix bug where client was noisy when scheduler closed unexpectedly (:pr:`1806`) `Matthew Rocklin`_
+-  Use string-based timedeltas (like ``'500 ms'``) everywhere `Matthew Rocklin`_
+-  Keep logs in scheduler and worker even if silenced (:pr:`1803`) `Matthew Rocklin`_
+-  Support minimum, maximum, wait_count keywords in Adaptive (:pr:`1797`) `Jacob Tomlinson`_ and `Matthew Rocklin`_
```

Comment:
Might be worth adding PR ( https://github.com/dask/distributed/pull/1800 ) here.